Gemfile

```diff
@@ -4,6 +4,10 @@ source 'https://rubygems.org'
 
 gem 'onlyoffice_digitalocean_wrapper'
 gem 'rake'
+gem 'net-ssh', '~> 6.1'
+gem 'net-sftp', '~> 2.1', '>= 2.1.2'
+gem 'ed25519', '>= 1.2', '< 2.0'
```

Comment:
I didn't find for which reason `ed25519` and `bcrypt_pbkdf` gems are required?

Reply:
I use the ed25519 key format to access remote servers. 
These libraries are needed to implement access via net-ssh because this format is still not supported.

https://bugzilla.redhat.com/show_bug.cgi?id=1747751

Reply:
Also information on the topic

https://github.com/net-ssh/net-ssh/issues/478